tkg/client/init.go

```diff
@@ -273,6 +273,20 @@ func (c *TkgClient) InitRegion(options *InitRegionOptions) error { //nolint:funl
 		targetClusterNamespace = options.Namespace
 	}
 
+	if config.IsFeatureActivated(constants.FeatureFlagPackageBasedCC) {
+		// for cc based cluster, check if tkr/cbt is reconciled to avoid package not found error by addons-controller
+		tkrVersion, err := c.TKGConfigReaderWriter().Get(constants.ConfigVariableTkrName)
```

Comment:
should we rename `tkrVersion` to `tkrName`?

Reply:
sure. Updated